_test_null_safety/mono_pkg.yaml

```diff
@@ -8,7 +8,9 @@ os:
 
 stages:
 - analyze_and_format:
-  - dartanalyzer: --fatal-infos .
+  - group:
+    - dartanalyzer: --fatal-infos
+    dart: dev
```

Comment:
We added a link that only comes up in THIS repo, which is...weird.

Reply:
Ya I don't know why we link in the top level analysis options here... but 🤷‍♂️